### ISSUE TITLE
[Request] Key spawn rate

### ISSUE THREAD

**Issue by MathiusXII** [BananaPuncher714/LethalQuantities#89]
Keys spawn separate from scrap right? Does this let you edit their spawn rate?

**Comment by BananaPuncher714** [comment]
Yes, the keys spawnrate is not tied with the scrap, although it should be able to be added as a separate scrap option. Normally, the way that the DunGen generation works is that there should always be one key for each locked door, but I'd have to take a look at how LethalCompany does it, since it seems to be different. I'll try to add an option if possible.
